### ISSUE TITLE
PixelDiscriminator should not use bias in Convolution layer with BatchNorm layer

### ISSUE THREAD

**Issue by kevinptt0323** [junyanz/pytorch-CycleGAN-and-pix2pix#695]
In `PixelDiscriminator` in `network.py`, the code about `use_bias` is incorrect.

The correct code should be same as the one in `NLayerDiscriminator`.

```python
        if type(norm_layer) == functools.partial:  # no need to use bias as BatchNorm2d has affine parameters
            use_bias = norm_layer.func != nn.BatchNorm2d
        else:
            use_bias = norm_layer != nn.BatchNorm2d
```

https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/8a6175415e57ce61e2ba35ff93507d7d54366430/models/networks.py#L598-L601

**Comment by junyanz** [comment]
Good catch. Just updated the code.
